pom.xml

```diff
@@ -352,98 +348,33 @@
-
                         <exlude>
                             <!-- Trello card for this exclude: https://trello.com/c/MCpbYlIT/1721-babbage-netty-multiple-vulnerabilities-s1 -->
-                            <!-- This is a transitive dependency of org.elasticsearch:elasticsearch:2.1.1 -->
+                            <!-- This is a transitive dependency of org.elasticsearch:elasticsearch:2.4.6 -->
                             <groupId>io.netty</groupId>
                             <artifactId>netty</artifactId>
-                            <version>3.10.5.Final</version>
+                            <version>3.10.6.Final</version>
                         </exlude>
-
-                        <exclude>
-                            <!-- Trello card for this exclude: https://trello.com/c/9HypMsIt -->
-                            <groupId>com.lowagie</groupId>
-                            <artifactId>itext</artifactId>
-                            <version>2.1.7</version>
-                        </exclude>
-
-                        <exclude>
-                            <!-- Trello card for this exclude: https://trello.com/c/ATXjzIlG/1005-upgrade-handlebars -->
-                            <groupId>com.github.jknack</groupId>
-                            <artifactId>handlebars</artifactId>
-                            <version>4.0.3</version>
-                        </exclude>
-                        <exclude>
-                            <!-- This is a transitive dependency of com.github.jknack:handlebars:4.0.3 -->
-                            <groupId>org.mozilla</groupId>
-                            <artifactId>rhino</artifactId>
-                            <version>1.7R4</version>
-                        </exclude>
-
-                        <exclude>
-                            <!-- Trello card for this exclude: https://trello.com/c/1XDM7LBH/1006-upgrade-rome-rome-dependency -->
-                            <groupId>rome</groupId>
-                            <artifactId>rome</artifactId>
-                            <version>1.0</version>
-                        </exclude>
-
                         <exclude>
                             <!-- https://trello.com/c/wotCeMmc -->
                             <groupId>org.yaml</groupId>
                             <artifactId>snakeyaml</artifactId>
-                            <version>1.33</version>
+                            <version>1.15</version>
                         </exclude>
-
-                        <exclude>
-                            <!-- This is to exclude vulnerability CVE-2023-35116 -->
-                            <groupId>com.fasterxml.jackson.core</groupId>
-                            <artifactId>jackson-databind</artifactId>
-                            <version>2.15.0</version>
-                        </exclude>
-
                         <exclude>
                             <!-- This is to exclude vulnerability CVE-2023-2976 -->
```

Comment:
Small one, but is this still true? After upgrading to v23, is this still here to exclude that vulnerability, or is it a new one?

Reply:
yes, unfortunately, v23 still has 3 vulnerabilities. I tried loading v33 first but then babbage throws an error related to handlebars again. Handlebars 4.2.1 requires  v23. I believe the next version of handlebars when they supported java 17  that is 4.3.1 will use a later version of guava. But this handlebar version would not work with babbage and java 8.

Reply:
So as soon they can update to handlebars 4.3.1 ONS would get this vulnerability solved. There might be a way to use this version in java 8 but I think it requires time for investigation. There are discussions about the issue in the web and the handlebars gitlab repo.